SeoPresentation.php

```diff
@@ -136,14 +136,9 @@ public function addExtractor(ExtractorInterface $extractor, $priority = 0)
     }
 
     /**
-     * Gets the SeoMetadata based on the content that contains the content.
-     *
-     * @param object $content
-     *
-     * @throws Exception\InvalidArgumentException
-     * @return SeoMetadata
+     * {@inheritDoc}
      */
-    private function getSeoMetadata($content)
+    public function getSeoMetadata($content)
```

Comment:
@dbu http://3v4l.org/ECNUM PHP does bother (not the other way around though).

And yeah, I can live with it too, but it has to be added to the CHANGELOG with a `[BC BREAK]` prefix imo